taiwan_bot_sheet.py

```diff
@@ -51,20 +52,24 @@ def __init__(self, context=SpreadsheetContext.GENERAL):
         _logger.info('Initiating TaiwanBotSheet')
         self.context = context
 
-    def get_questions_answers(self):
-        sheet = self.client.open(SPREADSHEET_FAQ_FILE).worksheet(CONTEXTS[self.context]["sheet"])
+    def get_questions_answers(self, context=None):
```

Comment:
wondering if that's necessary to pass  `context`? we can change to with `set_context(context)` instead?

Reply:
This is called only one time during the initialization, so I feel that this is kind of weird to call the function twice while calling `set_context` in between.

Also I tend to think that the FAQBot class should be the only one to manage the context, not the TaiwanBotSheet class which is more like a wrapper around our Google Sheets, it does not really need to have a notion of context.

Don't you think?

Reply:
I agree with Jonathan. The concept of the context is closely related to the conversation. Might make sense to keep it in this bot class for now since it's closely managing the conversation. 

In future when we have more contexts shared by different bots then maybe we can make an abstract context manager.

Reply:
👍 sounds good to me